### ISSUE TITLE
output binary image

### ISSUE THREAD

**Issue by 769cc** [qaz670756/LSNet#2]
Instead of outputting the binary image of the test set, it is directly compared with the standard result to obtain the accuracy. Can you tell me how to output the binary image of the test set？ Thank you.

**Comment by qaz670756** [comment]
I guess the 'output' you meant is Line126 in 'train.py':
`cd_preds = model(torch.cat([batch_img1,batch_img2],dim=0))`
It is then transformed into binary image in Line133-134:
`cd_preds = cd_preds[-1]`
`_, cd_preds = torch.max(cd_preds, 1)`
Because it's probability map with 2 channel, the first respresent the probability of unchanged areas while the second means change areas. So just compare them with 'torch.max' to output the binary image.

**Comment by 769cc** [comment]
If I want to use your model to run my test dataset and output binary graphs, but my dataset does not have label, it just have A and B， what should I do? Should I delete the label section of the dataloders？ thank you
